src/base/preferences.cpp

```diff
@@ -1611,10 +1642,9 @@ void Preferences::upgrade()
     QStringList labels = value("TransferListFilters/customLabels").toStringList();
     if (!labels.isEmpty()) {
         QVariantMap categories = value("BitTorrent/Session/Categories").toMap();
-        foreach (const QString &label, labels) {
+        foreach (const QString &label, labels)
```

Comment:
No, I meant that one can use range-based for here too, instead of Qt's foreach macro